remote_test.go

```diff
@@ -44,12 +44,55 @@ var (
 
 func TestMain(m *testing.M) {
 	flag.Parse()
+	err := setDriverPaths()
+	if err != nil {
+		fmt.Fprint(os.Stderr, fmt.Sprintf("Exiting early: unable to get the driver paths -- %s", err.Error()))
+		os.Exit(1)
+	}
+
 	s := httptest.NewServer(http.HandlerFunc(handler))
 	serverURL = s.URL
 	defer s.Close()
 	os.Exit(m.Run())
 }
 
+func setDriverPaths() error {
+	if strings.EqualFold(*selenium3Path, "") {
```

Comment:
Why use EqualFold and not just ==?

Reply:
It is a bad habit of mine. Anytime I compare strings, I use strings.EqualFold. However, in this case, there is no reason not to use ==. Will do